File: python/1036.py
valores = input().split()

valores = [float(i) for i in valores]

delta = valores[1]**2 - 4 * valores[0] * valores[2]

if delta < 0 or valores[0] == 0:
    print("Impossivel calcular")
else:
    R1 = (-valores[1] + delta**0.5) / (2 * valores[0])
    R2 = (-valores[1] - delta**0.5) / (2 * valores[0])

    print(f'R1 = {R1:.5f}')
    print(f'R2 = {R2:.5f}')

File: lua/1036.lua
local a = io.read("*n")
local b = io.read("*n")
local c = io.read("*n")

local delta = b^2 - 4*a*c

if delta < 0 or a == 0 then
    print("Impossivel calcular")
    return
end

local r1 = (-b + math.sqrt(delta)) / (2 * a)
local r2 = (-b - math.sqrt(delta)) / (2 * a)

print(string.format("R1 = %.5f", r1))
print(string.format("R2 = %.5f", r2))
